hack/verify-codegen.sh

```diff
@@ -3,7 +3,15 @@
 if [ "$IS_CONTAINER" != "" ]; then
   set -x
   go generate ./pkg/types/installconfig.go
+  rc=$?
+  if [ $rc != 0 ]; then
+      exit 1
+  fi
   go generate ./pkg/rhcos/ami.go
```

Comment:
This is going to fail because the file no longer exists. We need a replacement for this. But we can comment it out for now.